### DISCUSSION TITLE
[BUG] La sección Data Base debe mostrar sólo los elementos activos o tambien los inactivos?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#72] [4. Bug Reports] [answered]
Hola quisiera saber si es correcto lo siguiente con Data Base:

1.  Muestra las colecciones que uno quiere habilitar.
2.  Muestra todas los elementos existentes en la colección agregada (ya sea activos / inactivos)
3.  Muestra las subelementos de los elementos (activos / inactivos).

En el caso que es correcto esto: ¿Se puede hacer que muestre sólo aquellos que se encuentran activos?

**Fundamento:** Tengo una lista de categorías (A, B, C, D, E) y hago una mantención dejando: A, B, D, F, G (desactivé C, E y creé F, G). Entonces en data base me debe reflejar **A, B, D, F, G** y no **A, B, C, D, E, F, G** respectivamente. Si se muestran todos, puede que el cliente, a la hora de ocuparlo, se confunda.

Dejo un ss de Data Base de ejemplo visual.

![image](https://user-images.githubusercontent.com/61419471/114464507-d9c5c400-9bb3-11eb-9062-2fe100c2af56.png)

Gracias :>

**Comment by acedward** [comment, marked as answer]
@NicolasPLG Así es cómo debería funcionar: Solo activos deben aparecer.
 
Inactivos visibles, suena que es un bug @guillevalin

**Reply by NicolasPLG** [reply]
@acedward entiendo eddie gracias!. y si, por lo vemos cuando veo un elemento veo con varios desactivados. Lo levanto como un bug? o lo verá primera @guillevalin ? :D

**Reply by guillevalin** [reply]
@NicolasPLG por favor reporta un bug interno, solo deberían verse activos.

**Reply by NicolasPLG** [reply]
agrego: Revisé nuevamente haciendo un QA rápido antes de levantar el bug, con desactivar elementos y no me los refleja en el data (tanto como en la colección como subpropiedad de un elemento de la colección). Así que no lo crearé.  Gracias de nuevo

**Comment by NicolasPLG** [comment]
Ok lo reportaré gracias
